codex/conf.nim

```diff
@@ -227,6 +227,12 @@ type
         name: "eth-account"
       .}: Option[EthAddress]
 
+      ethPrivateKey* {.
```

Comment:
Should we really allow passing a private hey from the cmd? It seems very insecure as it gets logged in multiple place, like your history, or `ps -ef`, etc... I would suggest we use a file instead similar to https://github.com/codex-storage/nim-codex/blob/master/codex/utils/keyutils.nim

Reply:
Yes, good point. It wasn't meant to be the final solution, just something to get some testing going on an L2. But it shouldn't be too much effort to use a file instead.

Reply:
Fixed in 7017a91e1436c4e80fd208116345bccf48e8ec96